lib/elixir/lib/code.ex

```diff
@@ -2085,11 +2100,11 @@ defmodule Code do
 
   ## Examples
 
-      iex> Code.loaded?(Atom)
-      true
+      Code.loaded?(Atom)
```

Comment:
Please revert this and use a module that is likely to already be loaded, such as `Enum` or `String`. `Atom` indeed may not be loaded.